code/modules/admin/verbs/mapping.dm

```diff
@@ -30,6 +30,12 @@ var/intercom_range_display_status = 0
 		src.pixel_x = -224
 		src.pixel_y = -224
 
+/obj/effect/debugging/mapfix_marker

```

Comment:
It's to aid the mapper in fixing the conflicts, he won't have to hunt down for messed up tiles.